examples/python/guestbook/__main__.py

```diff
@@ -0,0 +1,183 @@
+# Copyright 2016-2018, Pulumi Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pulumi
+from pulumi_kubernetes.core.v1 import Service
+from pulumi_kubernetes.apps.v1 import Deployment
+
+redis_leader_labels = {
+    "app": "redis",
+    "tier": "backend",
+    "role": "master"
+}
+
+redis_leader_service = Service("redis-leader",
+    spec={
+        "ports": [{
+            "port": 6379,
+            "target_port": 6379,
+        }],
+        "selector": redis_leader_labels
+    },
+    metadata={
```

Comment:
`metadata` is typically defined before the `spec`.

Reply:
will do.